### ISSUE TITLE
Packet Loss

### ISSUE THREAD

**Issue by mindcoding** [DragonetMC/Dragonet-Legacy#32]
Is there an easy way to fix Packet loss?

**Comment by mindcoding** [comment]
Since I am inpatient Bump

**Comment by wordandahalf** [comment]
@mindcoding Packet loss = bad WiFi, we can't fix your Internet
